digdag-server/src/main/java/io/digdag/server/rs/ProjectResource.java

```diff
@@ -204,6 +205,8 @@ public ProjectResource(
         this.scsp = scsp;
         this.projectArchiveLoader = projectArchiveLoader;
         MAX_SESSIONS_PAGE_SIZE = systemConfig.get("api.max_sessions_page_size", Integer.class, DEFAULT_SESSIONS_PAGE_SIZE);
+        MAX_ARCHIVE_TOTAL_SIZE_LIMIT = systemConfig.get("api.max_archive_total_size_limit", Integer.class, DEFAULT_ARCHIVE_TOTAL_SIZE_LIMIT);
```

Comment:
@hiroyuki-sato is it possible to print debug log to print the value of MAX_ARCHIVE_TOTAL_SIZE_LIMIT because, otherwise, I was not sure my configuration was correctly effective or not.

Reply:
I agree with you. but users may want to know another parameter, for example, `max_attempts_page_size`.
It is better to create another issue for logging.
And I have no idea how to output that parameter after launch digdag.

@muga , @yoyama Do you have any idea?

Reply:
Agree > users may want to know another parameter, for example, `max_attempts_page_size`.

Another PR is fine.